templates/interfaces_nat.j2

```diff
@@ -12,3 +12,5 @@ iface lxcbr0 inet static
     # add checksum so that dhclient does not complain.
     # udp packets staying on the same host never have a checksum filled else
     post-up iptables -A POSTROUTING -t mangle -p udp --dport bootpc -s {{ lxcbr0_ip }}/{{ lxcbr0_netmask }} -j CHECKSUM --checksum-fill
+    post-up dnsmasq --pid-file=/var/run/lxc-dnsmasq.$IFACE.pid --conf-file=/etc/dnsmasq.$IFACE.conf
+    post-down kill $(cat /var/run/lxc-dnsmasq.$IFACE.pid) || true
```

Comment:
Not sure this is a good idea. You do not know if the dnsmasq instance is used for another usage. You should add a if statement IMHO.